src/memory-analyzer/memory_analyzer_parse_options.cpp

```diff
@@ -42,6 +42,13 @@ memory_analyzer_parse_optionst::memory_analyzer_parse_optionst(
 {
 }
 
+void memory_analyzer_parse_optionst::register_languages()
+{
+  // For now only C is supported due to the additional challenges of
+  // mapping source code to debugging symbols in other languages.
```

Comment:
Worth a reference to a GitHub issue here?

Reply:
I didn't know there was one!  I'll have a look...

Reply:
I'm not immediately finding one; if you had one in mind can you post a link?

Reply:
Apologies for the chasing of the geese that are wild  (positively livid...), I sort of half-assumed this was referring to the tickets discussing possible refactorings around `languaget` and modes - but I realise now I mis-read this comment.